### PR TITLE
Added Syft image

### PR DESCRIPTION
**Pull request by mackeyaj**
# PR Details

## Description
Added syft folder containing dockerfile, makefile, license and entry-point.sh script. This image supports the in development syft library step to generate a software bill of materials for container images.

## How Has This Been Tested

This image was pushed to the UIP image repo where its was used in conjunction with the sfyt library to generate a SBOM in the SDP pipeline.

## Types of Changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Documentation may be needed unsure of the doc reqs for adding new images.

### CODE REVIEW

File: syft/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# This software package is licensed under the Booz Allen Public License. The license can be found in the License file or at http://boozallen.github.io/licenses/bapl
+
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8/ubi
+ARG BASE_TAG=8.4
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+### Required Atomic/OpenShift Labels - https://github.com/projectatomic/ContainerApplicationGenericLabels
+LABEL name="Solutions Delivery Platform: Syft" \
+      maintainer="terrana_steven@bah.com" \
```
Comment (psig-bah) [inline syft/Dockerfile:11]: similar to #40, suggest updating to use your email, or a UIP mailbox

File: syft/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      vendor="Booz Allen Hamilton" \
+      version="0.1.0" \
+      release="0.1.0" \
```
Comment (psig-bah) [inline syft/Dockerfile:14]: change `version` and `release` to match application version ("0.36.0")

File: syft/LICENSE [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- **Limitation of Liability.** Under no circumstances and under no legal theory, whether tort (including negligence), contract, or otherwise, shall any Originator, or anyone who distributes the Product in accordance with this License, be liable to You for any direct, indirect, special, incidental, or consequential damages of any character including, without limitation, damages for lost profits, loss of goodwill, work stoppage, computer failure or malfunction, or any and all other commercial damages or losses, even if informed of the possibility of such damages.
+- **Severability.** If the application of any provision of this License to any particular facts or circumstances shall be held to be invalid or unenforceable, then the validity and enforceability of other provisions of this License shall not in any way be affected or impaired thereby.
```
Comment (psig-bah) [inline syft/LICENSE:101]: looks like this content got pasted in twice

**Review by psig-bah — CHANGES_REQUESTED** [review]
initial review: looks good! a few minor content changes requested, and I'll get started testing against the associated sdp-library

Thanks for submitting this and #40!

File: syft/Dockerfile (hunk 1 above)
Comment (mackeyaj) [reply syft/Dockerfile:11]: changed to uip distro

File: syft/LICENSE (hunk 3 above)
Comment (mackeyaj) [reply syft/LICENSE:101]: removed dupe content

**Review by psig-bah — APPROVED** [review]
looks good to me!

**Pull request merged**